modules/systemd/host.nix

```diff
@@ -0,0 +1,345 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  # Ghaf configuration flags
+  cfg = config.ghaf.systemd.host;
+  cfgDebug = config.ghaf.profiles.debug.enable;
+  cfgGui = config.ghaf.graphics.weston.enable;
+
+  # Override minimal with host configuration set
+  packageConfiguration = pkgs.systemdMinimal.override {
+    pname = "host-systemd";
+    withAcl = true;
+    withAnalyze = cfg.withAnalyze;
+    withAudit = cfg.withAudit;
+    withCompression = cfgDebug || cfg.withVirtualization;
```

Comment:
Why disable compression of journal file blocks at all? Maybe we should also think what do we do with systemd-journalling on release-build?

Reply:
Good point - was only thinking of coredump compression at the time, and also was unsure about how to deal with journaling in this configuration, as I anticipated we currently need it even in release. 
Let's discuss a strategy for that - probably we want encrypted logs on release, and auditd by default for the time being.